browser/brave_browser_process_impl.cc

```diff
@@ -212,6 +214,11 @@ void BraveBrowserProcessImpl::StartBraveServices() {
     https_upgrade_exceptions_service();
   }
 
+  if (base::FeatureList::IsEnabled(
+          brave_shields::features::kBraveLocalhostAccessPermission)) {
+    localhost_permission_allowlist_service();
```

Comment:
but it will be created in `OnBeforeURLRequest_LocalhostPermissionWork` anyway?

Reply:
Would you suggest not creating it here then? Or DCHECKing in `OnBeforeURLRequest_LocalhostPermissionWork` that the service is created, and not trying to create it there?

Reply:
no, I mean we can create, but here it is guarded with the feature and in `OnBeforeURLRequest_LocalhostPermissionWork` it wasn't

Reply:
I create a new one in `OnBeforeURLRequest_LocalhostPermissionWork` now - makes sense to be consistent.